docs/updates/2019-02.md

```diff
@@ -0,0 +1,93 @@
+# February 2019
+
+Welcome to the February 2019 Qlik Core update. We're highlighting a couple of important changes and
+improvements which we hope you will like.
+
+## Usability and design changes in catwalk
+
+Since being moved to stable last month, we've made several small usability improvements and design changes along with a
+couple of bug fixes.
+
+### QlikView select excluded coloring
+
+We have added support for the *select excluded* selection state, which can be used for deeper data model analysis. A
+value enters the selected excluded state because the value was first selected, and then excluded by a selection in
+another field. This value will be described in catwalk with *green on light grey background*.
+
+!!! Note
+    Please note the color scheme follows QlikView coloring and corresponds to *dark grey and check mark* in Qlik
+    Sense.
+
+More information about Qlik's selection model can be found in the
+[Qlik Sense help pages](https://help.qlik.com/en-US/sense/Subsystems/Hub/Content/Sense_Hub/Selections/associative-selection-model.htm).
+
+![screenshot](../images/select-excluded.gif)
+
+### Reworked top bar with new menu
+
+The top bar in catwalk has been reworked, adding a new menu containing project-wide links. Apart from the existing link
+to the guide, you can also navigate to the Engine documents overview through *Choose App*.
+
+![screenshot](../images/catwalk-top-bar.png)
+
+### Guide for Qlik Sense Enterprise integration
+
+Since a lot of our users come from a Qlik Sense background, we have created a
+[guide](https://github.com/qlik-oss/catwalk#qlik-sense-enterprise) which describes how to open documents from Qlik Sense
```

Comment:
Use another word than 'guide' here. Guide is used above as 'The guide' :)
Maybe sth like 'a how-to has been added to the readme ..'